File: ApiTemplate/WebApplication1/DomainServices/Entities/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebApplication1.DomainServices.Entities
{
    public class Person : EntityBase
    {
        public int DocumentType { get; set; }
        public string DocumentNumber { get; set; }
        public string FirstName { get; set; }
        public string SecondName { get; set; }
        public string FirstLastName { get; set; }
        public string SecondLastName { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public string Address { get; set; }
        public int RoleId { get; set; }
    }
}

File: client/src/app/shared/interfaces/person.ts
export interface Person {
  id: number;
  documentType: number;
  documentNumber: string;
  firstName: string;
  secondName: string;
  lastName: string;
  secondLastName: string;
  email: string;
  phone: string;
  address: string;
  roleId: number;
  password: string;
  userChange: number;
  dateModified: Date;
}

export interface DocumentType {
  id: number;
  resume: string;
  name: string;
  description: string;
  userchange: number;
  datemodified: Date;
}
